master/internal/command/command.go

```diff
@@ -74,6 +74,7 @@ type command struct {
 	readinessMessageSent bool
 	metadata             map[string]interface{}
 	serviceAddress       *string
+	assignedPort         int
```

Comment:
since 0 port can sometimes mean "auto assign a port" in port world, can we use the less ambiguous *int for the port.

Reply:
done